### ISSUE TITLE
script injection is possible

### ISSUE THREAD

**Issue by wallcrawler** [dwyl/hapi-socketio-redis-chat-example#15]
this could help
```
function sanitise(txt) {
  if(txt.indexOf("<") > -1 || txt.indexOf(">") > -1) {
    txt = txt.replace(/</g, "&lt").replace(/>/g, "&gt");
  }
  return txt;
}
```

**Comment by nelsonic** [comment]
Well spotted!
Thanks for reviewing the code!
Agree 100%!
We don't have _any_ escaping so there is potential for XSS.
If you have time to submit a PR please do! Thanks!

**Comment by nelsonic** [comment]
Fixed by #16
